### ISSUE TITLE
🆕 Software Suggestion | privacy-redirect: redirects Twitter, YouTube, Instagram, Google Maps, Reddit & Google Search requests to privacy friendly alternatives

### ISSUE THREAD

**Issue by GETandSELECT** [privacytools/privacytools.io#2187]
I like very much https://github.com/SimonBrazell/privacy-redirect (I am only a random user)
See also privacy policy: https://github.com/SimonBrazell/privacy-redirect/wiki/Privacy-Policy

> A web extension that redirects Twitter, YouTube, Instagram, Google Maps, Reddit & Google Search requests to privacy friendly alternatives - Nitter, Invidious, FreeTube, Bibliogram, OpenStreetMap & Private Search Engines like DuckDuckGo and Startpage.
> 
> It's possible to toggle all redirects on and off. The extension will default to using random instances if none are selected. If these instances are not working, you can try and set a custom instance from the list below.

I wish to discuss here first, before I make a PR for adding this addon to the Firefox page.
What you think?

**Comment by freddy-m** [comment]
On the whole, the fewer addons you have the better. So I'm not in favour of adding this as an addon. That being said, I think these products (Nitter, Invidious, Teddit, Bibliogram) should be added in their respective sections.

**Comment by LongJohn-Silver** [comment]
> On the whole, the fewer addons you have the better. So I'm not in favour of adding this as an addon. That being said, I think these products (Nitter, Invidious, Teddit, Bibliogram) should be added in their respective sections.

What about something like violent monkey which has other features but for which a script could be written to replace this?

**Comment by wylel** [comment]
I think this could be added to the Additional Functionality section.

**Comment by AnaemicStar** [comment]
> On the whole, the fewer addons you have the better. So I'm not in favour of adding this as an addon. That being said, I think these products (Nitter, Invidious, Teddit, Bibliogram) should be added in their respective sections.

Yes, I agree the fewer Add-ons on your browser the better. Privacy Redirect seems good though. Recommending Nitter, Invidious, Llibreddit and Bibliogram on Privacy Tools would be a great idea.

**Comment by GrimPixel** [comment]
I think it could be necessarily convenient when the search results are often provided as those sites to be redirected.
